src/Matroids/properties.jl

```diff
@@ -0,0 +1,302 @@
+	for A in sets
+		for B in sets
+			if issubset(A,B) ⊻ issubset(B,A)
+				return false
+			end
+		end
+	end
+	return true
+end
+
+function get_blocker(clutter::Union{AbstractVector{<:AbstractVector}, AbstractVector{<:AbstractSet}}, groundset::Vector)
+	#TODO
+end
+
+is_regular(M::Matroid) = M.pm_matroid.REGULAR
+
+is_graphic(M::Matroid) = true #TODO
+
+is_cographic(M::Matroid) = is_graphic(dual_matroid(M))
+
+is_binary(M::Matroid) = M.pm_matroid.BINARY
+
+is_ternary(M::Matroid) = M.pm_matroid.TERNARY
+
+n_connected_components(M::Matroid) = size(M.pm_matroid.CONNECTED_COMPONENTS)[1]
+
+connected_components(M::Matroid) = [[M.groundset[i+1] for i in comp] for comp in M.pm_matroid.CONNECTED_COMPONENTS]
+is_connected(M::Matroid) = M.pm_matroid.CONNECTED
+
+loops(M::Matroid) = [M.groundset[i+1] for i in M.pm_matroid.LOOPS]
+
+coloops(M::Matroid) = [M.groundset[i+1] for i in M.pm_matroid.DUAL.LOOPS]
+
+is_loopless(M::Matroid) = length(M.pm_matroid.LOOPS)==0 ? true : false
+
+is_coloopless(M::Matroid) = length(M.pm_matroid.DUAL.LOOPS)==0 ? true : false
+
+function direct_sum_components(M::Matroid)
+	res = Vector{Matroid}()
+	for set in connected_components(M)
+		res = [res; restriction(M,set)]
+	end
+	return res
+end
+
+connectivity_function(M::Matroid, set::Union{AbstractVector, Set}) = rank(M,set) + rank(M,setdiff( Set(M.groundset), set)) - rank(M)
+
+is_vertical_k_separation(M::Matroid,k::Int, set::Union{AbstractVector,Set}) = k<=rank(M,set) && k<=rank(M,setdiff( Set(M.groundset), set)) && k> connectivity_function(M,set)
+
+is_k_separation(M::Matroid,k::Int, set::Union{AbstractVector,Set}) = k<=size(set)[1] && k<=size(setdiff( Set(M.groundset), set))[1] && k> connectivity_function(M,set)
+
+function vertical_connectivity(M::Matroid)
+	gs = Set(M.groundset)
+	res = rank(M)
+	for set in cocircuits(M)
+		comp = setdiff(gs,set)
+		if length(set)==0 || length(set)>length(comp)
+			continue
+		end
+		rk_set = rank(M,set)
+		rk_comp = rank(M,comp)
+		k = minimum([rk_set,rk_comp])
+		if k<res && k>rk_set+rk_comp-rank(M)
+			res = k
+		end
+	end
+	return res
+end
+
+girth(M::Matroid, set::Vector=M.groundset) = minimum([inf; [issubset(C,set) ? size(C)[1] : inf for C in circuits(M)]])
+
+
+function tutte_connectivity(M::Matroid)
+	r = M.pm_matroid.RANK
+	n = M.pm_matroid.N_ELEMENTS
+	#if M is uniform, apply Cor. 8.6.3 otherwise Thm. 8.6.4
+	if n>2r && M.pm_matroid.N_BASES==binomial(n,r)
+		return n>=2r-2 ? r+1 : inf
+	end
+	return minimum([vertical_connectivity(M),girth(M)])
+end
+
+principal_extension()=0
+free_extension()=0
+series_extension()=0
+
+function tutte_polynomial(M::Matroid)
+	R, (x, y) = PolynomialRing(ZZ, ["x", "y"])
+	poly = M.pm_matroid.TUTTE_POLYNOMIAL
+	exp = Polymake.monomials_as_matrix(poly)
+	return R(Vector{Int}(Polymake.coefficients_as_vector(poly)),[[exp[i,1],exp[i,2]] for i in 1:size(exp)[1]])
+end
+
+function characteristic_polynomial(M::Matroid)
```

Comment:
Elsewhere we call this `charpoly`.

Reply:
Well, with "this" I mean "functions computing characteristic polynomials".

Of courser we might want to offer this long version as an alias name, as `charpoly` does not quite fit our current conventions for function names (where `characteristic_polynomial` would indeed fit better...). Dunno... Maybe @fieker or @thofma have something to say.

Reply:
I think it would be best to try to be consistent first and then we can add/switch to the the long version later in one swoop.